composer.json

```diff
@@ -10,7 +10,9 @@
         "ext-curl": "*",
         "ext-json": "*",
         "php": ">=7.2",
-        "psr/log": "^1.1"
+        "psr/log": "^1.1",
+        "edamov/pushok": "^0.14.0",
```

Comment:
Albeit small, this seems the go-to PHP library when it comes to apple push notifications :+1: . I was surprised no other bigger vendors provided this functionality.